runtime/include/qio/deque.h

```diff
@@ -45,8 +45,8 @@ extern "C" {
 
 #ifndef CHPL_RT_UNIT_TEST
 #include "chpl-mem.h"
-#define deque_calloc(nmemb, size) chpl_mem_allocManyZero(nmemb, size, CHPL_RT_MD_IO_BUFFER, __LINE__, __FILE__)
-#define deque_free(ptr) chpl_mem_free(ptr, __LINE__, __FILE__)
+#define deque_calloc(nmemb, size) chpl_mem_allocManyZero(nmemb, size, CHPL_RT_MD_IO_BUFFER, 0, 0)
```

Comment:
On second thought, maybe we could get everything we need here just by making sure the message texts were unique!